src/Illuminate/Container/Container.php

```diff
@@ -162,6 +162,17 @@ public function isAlias($name)
     }
 
     /**
+     * Determine if a given string is an alias abstract.
+     *
+     * @param  string  $name
+     * @return bool
+     */
+    public function isAliasAbstract($name)
+    {
+        return isset($this->aliasAbstracts[$name]);
```

Comment:
I suppose we could make the function private and reflect it during testing but that feels really, really wrong.

I'll see if I can come up with a test that fails or could in the future after a refactoring if the lists are not kept in sync.